modules/kubernetes-addons/cloudnative-pg/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "helm_config" {
+  description = "Helm provider config for the airflow."
```

Comment:
Update the description with Cloudnative PG Operator config and remove references to airflow